hook/filter/filter.go

```diff
@@ -22,6 +22,23 @@ var (
 		"/usr/lib/python2.7/lib-dynload/Python-2.7.egg-info",
 		"/usr/lib/python2.7/wsgiref.egg-info",
 	}
+
+	affectedTypes = []string{
+		//ruby
+		types.Bundler,
+		types.GemSpec,
+
+		//python
+		types.Pip,
+		types.Pipenv,
+		types.Poetry,
+		types.PythonPkg,
+
+		//node.js
+		types.NodePkg,
+		types.Npm,
+		types.Yarn,
```

Comment:
The target is individual packages, not lock files.

```suggestion
		// ruby
		types.GemSpec,

		// python
		types.PythonPkg,

		// node.js
		types.NodePkg,
```